package/MDAnalysis/core/universe.py

```diff
@@ -1523,6 +1526,13 @@ def guess_TopologyAttrs(
             TopologyAttr does not already exist in the Universe, this has no
             effect. If the TopologyAttr does already exist, all values will
             be overwritten by guessed values.
+        error_if_missing: bool
```

Comment:
add `.. versionadded:: 2.8.0`

Reply:
The method itself has a `versionadded:: 2.8.0` tag as it's new in 2.8.0 -- is that sufficient?

Reply:
Yes! Sorry didn’t realize.